content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx

```diff
@@ -1,17 +1,39 @@
 ---
 title: 'Examples'
-metaTitle: 'Prisma Client Extensions | Examples'
-metaDescription: 'Extend the functionality of Prisma Client: Examples'
+metaTitle: 'Prisma Client extensions | Prisma & Community-built extensions and examples'
+metaDescription: 'Prisma Client extensions | Prisma & Community-built extensions and examples'
 ---
 
-<TopBlock>
+## Extensions made by Prisma
 
-The extensions are provided as examples only, and without warranty. We recommend using the examples as a source of inspiration for building your own extensions.
+The following is a list of extensions we've built at Prisma:
+
+| Extension                                                                                             | Description                                                                                                               |
+| :---------------------------------------------------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+| [`@prisma/extension-accelerate`](https://www.prisma.io/docs/data-platform/accelerate/getting-started) | Enables Accelerate, a global database cache available in 300+ locations with built-in connection pooling                                                                           |
+| [`@prisma/extension-pulse`](https://www.prisma.io/docs/data-platform/pulse/what-is-pulse)             | Enables Pulse, a service that captures change events from your database and delivers them instantly to your applications. |
```

Comment:
are the repos for the extensions public? if so, could we link the `@prisma/extension-pulse` to the repo and and then link to the docs from the "product name"?

```suggestion
| [`@prisma/extension-pulse`](link-to-github)             | Enables [Pulse](https://www.prisma.io/docs/data-platform/pulse/what-is-pulse), a service that captures change events from your database and delivers them instantly to your applications. |
```

Reply:
Even if not, "Accelerate" and "Pulse" should definitely link somewhere useful.

Reply:
The repositories are private. I'll link to the product pages instead.

Reply:
linking to npm is maybe an option as well?